recipes/memray/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "memray" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 01fbbf43d434f81bed4d9e63f1e8eebcd9686945211b4cd4e51b2f03cc1004f5
+
+build:
+  skip: True  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+  host:
+    - python
+    - pip
+    - libunwind
+    - cython
+  run:
+    - python
+    - rich <11.0.0
+    - jinja2
```

According to https://github.com/bloomberg/memray/blob/main/setup.py#L85 also
```suggestion
    - jinja2
    - typing-extensions  # [py<38]
```